OpenRA.Game/ModMetadata.cs

```diff
@@ -46,6 +47,9 @@ public class ModMetadata
 				var mod = FieldLoader.Load<ModMetadata>(nd["Metadata"]);
 				mod.Id = m;
 
+				if (nd.ContainsKey("ContentInstaller"))
+					mod.Content = FieldLoader.Load<ContentInstaller>(nd["ContentInstaller"]);
```

Comment:
It might pay to wrap this in a try/catch.  You don't want to kill the whole game if someone has one bogus mod installed.